src/js/components/CandidateListRoot/CandidateListRoot.jsx

```diff
@@ -342,22 +339,27 @@ class CandidateListRoot extends Component {
         {(!(isSearching && hideDisplayBecauseNoSearchResults)) && (
           <CampaignsScrollingOuterWrapper>
             <LeftArrowOuterWrapper className="u-show-desktop-tablet">
-              <LeftArrowInnerWrapper onClick={() => {this.handleHorizontalScroll(this.scrollElement.current, 30, 621, -16)}}>
-                {this.state.hideLeftArrow ? null : <ArrowBackIos classes={{ root: classes.arrowRoot }} />}
+              <LeftArrowInnerWrapper onClick={() => { handleHorizontalScroll(this.scrollElement.current, 5, 621, -40, leftAndRightArrowStateCalculation); }}>
```

Comment:
Hi @rileychou great work on this pull request! I am going to approve this pull request, but when you get a chance, please create a new pull request regarding any of the comments I make. On this particular line, it is best practice to assign these variable numbers to variables that describe what the number means, like  `const horizontalScrollSpeed = 5;` and `const horizontalScrollDistance = 621;`

I think you want to pass in `this.checkScrollPositionLocal` so you can set the local variables hideLeftArrow and hideRightArrow, instead of the function that just does the calculation `leftAndRightArrowStateCalculated`, no?